runtime/jcl/common/java_lang_invoke_MethodHandleNatives.cpp

```diff
@@ -96,8 +96,17 @@ initImpl(J9VMThread *currentThread, j9object_t membernameObject, j9object_t refO
 
 	if (refClass == J9VMJAVALANGREFLECTFIELD(vm)) {
 		J9JNIFieldID *fieldID = vm->reflectFunctions.idFromFieldObject(currentThread, NULL, refObject);
+		J9ROMFieldShape *romField = fieldID->field;
+		UDATA offset = fieldID->offset;
+
+		if (J9_ARE_ANY_BITS_SET(romField->modifiers, J9AccStatic)) {
+			offset = fieldID->offset | J9_SUN_STATIC_FIELD_OFFSET_TAG;
```

Comment:
can this just be `offset |= J9_SUN_STATIC_FIELD_OFFSET_TAG;`

Reply:
fixed